components/automate-ui/src/app/services/projects-filter/projects-filter.selectors.ts

```diff
@@ -8,7 +8,7 @@ export const projectsFilterState = state => state.projectsFilter;
 export const options = createSelector(projectsFilterState, state => state.options);
 
 export const assignableProjects = createSelector(projectsFilterState, state => {
-  let projectOptions = state.options.filter((p: ProjectsFilterOption) => p.checked);
+  let projectOptions = (state.options as ProjectsFilterOption[]).filter(p => p.checked);
```

Comment:
Introduce strong type at the start so everything else in this block is now typed, too.